### PR TITLE
Fix more typos

### PR DESCRIPTION
**Pull request by eupn**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Replace gpio traits with digital::v2
 - Bump `stm32f1` dependency (`0.8.0`)
 - ADC now requires the clock configuration for initialisation
-- `disable_jtag` now transforms PA15, PB3 and PB4 to forbid their use without desactivating JTAG
+- `disable_jtag` now transforms PA15, PB3 and PB4 to forbid their use without deactivating the JTAG
```
Comment (TeXitoi) [inline CHANGELOG.md]: Not sure for this one.

Comment (TheZoq2) [reply CHANGELOG.md]: I think "the JTAG" sounds better though I'm not sure if it is technically correct

Comment (therealprof) [reply CHANGELOG.md]: I think it's just JTAG or the JTAG interface/function

Comment (TeXitoi) [reply CHANGELOG.md]: Let's just write "the JTAG interface" then.

Comment (eupn) [reply CHANGELOG.md]: Removed `the`

**Comment by TheZoq2** [comment]
Thanks!

**Pull request merged**